pkg/proxy/iptables/proxier.go

```diff
@@ -821,19 +818,15 @@ func (proxier *Proxier) syncProxyRules() {
 			if local, err := utilproxy.IsLocalIP(externalIP); err != nil {
 				glog.Errorf("can't determine if IP is local, assuming not: %v", err)
 			} else if local {
-				lp := utilproxy.LocalPort{
-					Description: "externalIP for " + svcNameString,
-					IP:          externalIP,
-					Port:        svcInfo.Port,
-					Protocol:    protocol,
-				}
-				if proxier.portsMap[lp] != nil {
-					glog.V(4).Infof("Port %s was open before and is still needed", lp.String())
-					replacementPortsMap[lp] = proxier.portsMap[lp]
+				desc := "externalIP for " + svcNameString
+				lpStr := utilnet.ToLocalPortString(externalIP, svcInfo.Port, protocol, desc)
```

Comment:
no need for a special function here - it doesn't have to be consistent across modules